pkg/components/managementconsole/component.go

```diff
@@ -22,6 +22,49 @@ var ManagementConsoleComponent = Component{
 			{
 				IncludeAll: []string{"bz-Management Console"},
```

Comment:
QE test cases all have string `UserInterface`
can we have a new `IncludeAll` matcher? 
```
			{
				IncludeAll: []string{"UserInterface"},
			},
```

Reply:
then we can safely remove all `{Suite}` matching lines from 25 ~ 67